core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -1606,4 +1661,26 @@ public QualifiedObjectName getName()
             return columns;
         }
     }
+
+    public static class QuantifierRange
```

Comment:
Maybe just call this `Range`. `QuantifierRange` looks funny, especially since we have a class named `RangeQuantifier`